app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -596,6 +603,54 @@ class BrowserTabFragment :
         addTabsObserver()
     }
 
+    private fun processFileDownloadedCommand(command: FileDownloadCallback.DownloadCommand) {
+        when (command) {
+            is FileDownloadCallback.DownloadCommand.ShowDownloadStartedMessage -> downloadStarted(command)
+            is FileDownloadCallback.DownloadCommand.ShowDownloadFailedMessage -> downloadFailed(command)
+            is FileDownloadCallback.DownloadCommand.ShowDownloadSuccessMessage -> downloadSucceeded(command)
+        }
+    }
+
+    private fun downloadStarted(command: FileDownloadCallback.DownloadCommand.ShowDownloadStartedMessage) {
+        if (command.showNotification) {
+            fileDownloadNotificationManager.showDownloadInProgressNotification()
+        }
+        view?.makeSnackbarWithNoBottomInset(getString(command.messageId, command.fileName), 750)?.show()
```

Comment:
These magic numbers (750, 1500 below) were moved to constants when refactoring to use modules (the other PR). 
```
const val DOWNLOAD_SNACKBAR_LENGTH = 750
const val DOWNLOAD_SNACKBAR_DELAY = 1500L
```